state/store/reducer.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import sessionReducer from '../session/reducer';
+import trendingMoviesReducer from '../movies/reducer';
 
 const reducer = combineReducers({
-  sessionReducer
+  sessionReducer,
+  trendingMoviesReducer
```

Comment:
It's better to set key without reducer word, because this key used as key in the result state. Like:
```
session: sessionReducer,
trendingMovies: trendingMoviesReducer
```

Reply:
Okay. Fixed.